catppuccinpalette.dtx

```diff
@@ -136,9 +136,13 @@
 %   \verb|pagecolor| &
 %   set the pagecolor of the document to \verb|CtpBackground|. Possible values:
 %   \verb|false| (default+initial), \verb|true|
```

Comment:
Note that this is not quite right I just noticed. The default (if no value is passed, but the option is given) is `true` for all boolean options.

I'll adjust this in another PR in the future

Reply:
See #22